platforms/ios/src/TangramMap/TGMapViewController+Internal.h

```diff
@@ -20,7 +20,9 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)clearDataSource:(std::shared_ptr<Tangram::TileSource>)tileSource;
 
-- (Tangram::SceneReadyCallback)setSceneReadyListener;
+- (Tangram::SceneReadyCallback)sceneReadyListener;
+
+- (std::vector<Tangram::SceneUpdate>)unpackSceneUpdates:(NSArray<TGSceneUpdate *> *)sceneUpdates;
```

Comment:
No need to add those to the internal header if they are not needed anywhere else than in the implementation file. They should be defined [here](https://github.com/tangrams/tangram-es/blob/89988a8241870a050bd4f8594b927b507ffc75aa/platforms/ios/src/TangramMap/TGMapViewController.mm#L26) instead.